src/v/redpanda/admin_server.cc

```diff
@@ -3446,6 +3446,20 @@ void admin_server::register_debug_routes() {
               [] { return ss::json::json_return_type(ss::json::json_void()); });
       });
 
+    register_route<user>(
+      ss::httpd::debug_json::refresh_disk_health_info,
+      [this](std::unique_ptr<ss::httpd::request>) {
+          vlog(logger.info, "Request to refresh disk health info");
+          return _metadata_cache.local().refresh_health_monitor().then_wrapped(
+            [](ss::future<> f) {
+                if (f.failed()) {
+                    throw ss::httpd::server_error_exception(
+                      "failed to refresh disk health info");
+                }
```

Comment:
1. when `f.failed()` is true then it is an exceptional future. however, here it is ignored. this will result in seastar throwing an exception declaring `seastar - Exceptional future ignored:`. typically this is solved by logging a note about the exception (which has the side effect of not ignoring the failed future), making seastar happy. i'd log some info, but not expose those internal issues through the server error that will be returned to the client.

2. since we aren't in a coroutine context here, we should be using ss::make_exceptional_future rather than throwing.